src/core/ext/filters/client_channel/lb_policy/priority/priority.cc

```diff
@@ -636,6 +636,7 @@ void PriorityLb::ChildPriority::StartFailoverTimerLocked() {
             priority_policy_->child_failover_timeout_ms_);
   }
   Ref(DEBUG_LOCATION, "ChildPriority+OnFailoverTimerLocked").release();
+  ExecCtx::Get()->InvalidateNow();
```

Comment:
This is not a repeated timer, so there can't be a loop here.

Reply:
removed